server/testdb/tests_test.go

```diff
@@ -81,3 +81,30 @@ func TestGetTests(t *testing.T) {
 	require.NoError(t, err)
 	assert.Len(t, actual, 0)
 }
+
+func TestGetTestsWithMultipleVersions(t *testing.T) {
+	db, clean := getDB()
+	defer clean()
+
+	test1 := createTestWithName(t, db, "1")
+	test1.Name = "1 v2"
+	test1.Version = 2
```

Comment:
`db.UpdateTest` should automatically increase the version, so we don't forget to do it manually in every place we update a test

Reply:
The thing is that we use `UpdateTest` internally. So I decided to introduce another method: `CreateTestVersion` that creates a new Test but doesn't prepopulate its fields like `ID`, just like `CreateTest` does.

Reply:
Still, [this part](https://github.com/kubeshop/tracetest/pull/528/files#diff-cdf13307c512afb9b589e95ccb04f91bc209833689b1563b5f363a9f7513e0feR289) needs to be encapsulated.

What if we had a `ForceUpdateTest` that just updates the test leaving the version untouched, and the `UpdateTest` that creates a new version?

Reply:
I think we can do that, I just don't like the `ForceUpdateTest` because it gives the impression that we have some rule that doesn't allow the test to be updated, and using that, we skip that rule.

What if we have `UpdateTest` and `UpdateTestVersion`? 
`UpdateTest` updates the test and creates a new version. `UpdateTestVersion` updates the details of a test version without creating a new one.

Reply:
yea, that looks much better. I like it

Reply:
Just pushed the change

Reply:
Forgot to remove the `test.Version = 2`. I'm gonna push this change now.